### PR TITLE
New Crowdin translations by Github Action

### PR DESCRIPTION
**Pull request by ss-o**
CodeSandbox logoCodeSandbox logo&nbsp; Open in CodeSandbox <a href="https://codesandbox.io/p/github/z-shell/wiki/l10n_crowdin_action?workspace=%7B%22gitSidebarPanel%22:%22PR%22,%22sidebarPanel%22:%22GIT%22%7D">Web Editor</a> | <a href="https://codesandbox.io/p/vscode?owner=z-shell&repo=wiki&branch=l10n_crowdin_action">VS Code</a>

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/current/intro.md [hunk 1]
```diff
@@ -4,17 +4,14 @@ slug: /
...
-  - about zi
+  - caractéristiques de zi
+  - à propos de zi
 ---
 
-import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
-import Link from '@docusaurus/Link';
-import Image from '@theme/IdealImage';
-import ZGitImg from '@site/static/img/png/z_git.png';
+import Tabs de '@theme/Tabs' ; import TabItem de '@theme/TabItem' ; import Link de '@docusaurus/Link' ; import Image de '@theme/IdealImage' ; import ZGitImg de '@site/static/img/png/z_git.png' ;
```
Comment (ss-o) [inline i18n/fr/docusaurus-plugin-content-docs/current/intro.md:14]: @0xMRTT,

You reformated the file, need to fix it manually and upload or directly in UI.
Always use: "Hide string" for everything that doesn't need to be translated.

**Comment by ss-o** [comment]
Disabled workflow to prevent conflicts.

- [x] https://github.com/z-shell/wiki/actions/workflows/crowdin-download.yml

**Comment by ss-o** [comment]
Still, fixing this one?

**Comment by 0xMRTT** [comment]
> Still, fixing this one?

When I go to the workflow, I see that the workflow is disable. What can I do more ?

**Comment by ss-o** [comment]
It's the disabled workflow to prevent conflicts not to translate and prevent adverse effects on translation memory, etc. until you fix https://github.com/z-shell/wiki/pull/18/files/834f912677265cfadde9bd48a8a0a4c7fcc7cc78#diff-a58e39cde8d0fbbd09e7928cfae79eb2964d325ae04b989b52c9cf610c8fde71.

**Review by 0xMRTT — DISMISSED** [review]

**Comment by ss-o** [comment]
I am confused, What are you approving and why? It is broken due translation of the script. Do you need help?

**Comment by 0xMRTT** [comment]
> I am confused, What are you approving and why? It is broken due translation of the script. Do you need help?

Sorry 😔. I understand where is the error. 

But why crowdin add this to the translation?😅

**Comment by ss-o** [comment]
Because you have access to approve, I explained already in a comment. It's not a problem, but better to do it all without fixing stuff as costs time. The error fix of 5 min. actually took a day also had to explain it multiple times on error which should not have to go through in the first place. Maybe you need to check https://support.crowdin.com/translation-process-overview/ to improve and avoid such scenarios.

> It is not a problem for me to explain it multiple times when something is planned.

**Review by 0xMRTT — COMMENTED** [review]
It's now fixed on crowdin

**Comment by ss-o** [comment]
Thanks, I'll do the rest :octocat: :+1:

File: i18n/fr/docusaurus-plugin-content-docs/current/intro.md (hunk 1 above)
Comment (ss-o) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md:14]: Closed to fetch new files.

Comment (0xMRTT) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md:14]: Thanks. I've updated on crowdin and I will be more attentive for the rest 🙂

**Pull request closed without merging**